fetch.bs

```diff
@@ -49,6 +49,10 @@ urlPrefix:https://w3c.github.io/hr-time/#;spec:hr-time
         url:dfn-unsafe-shared-current-time;text:unsafe shared current time
     type:typedef;url:dom-domhighrestimestamp;text:DOMHighResTimeStamp
 
+urlPrefix:https://w3c.github.io/webappsec-permissions-policy;spec:permissions-policy
+    type:dfn
+        url:is-feature-enabled;text:is feature enabled in document for origin?
```

Comment:
This should be exported. Surprised it isn't already.

Reply:
It is exported, but I can't get bikeshed to recognize it without this line. Am I holding it wrong?